src/react-jw-player.jsx

```diff
@@ -64,34 +66,36 @@ class ReactJWPlayer extends Component {
     return hasFileChanged || hasPlaylistChanged;
   }
   componentDidUpdate() {
-    if (window.jwplayer && window.jwplayer(this.props.playerId)) {
+    if (window.jwplayer && window.jwplayer(this.videoRef)) {
       this._initialize();
     }
   }
   componentWillUnmount() {
-    removeJWPlayerInstance(this.props.playerId, window);
+    removeJWPlayerInstance(this.videoRef, window);
   }
   _initialize() {
     const { playerId, useMultiplePlayerScripts } = this.props;
 
+    this.videoRef.id = playerId;
```

Comment:
Any specific reason to set this here rather than attaching the id to the element directly in the render method?

Reply:
No, edited this